### DISCUSSION TITLE
Suport for Angular 17 server side rendering

### DISCUSSION THREAD

**Discussion by ek1mb4** [Azure/static-web-apps#1361] [General]
This seems like a great thing, so I would like to know if there is support for Angular SSR.

**Comment by thomasgauvin** [comment]
Thanks for the suggestion @ek1mb4, see my response to https://github.com/Azure/static-web-apps/discussions/1362. It's not currently supported as it requires an Express server but we are aware of this new functionality and actively looking into opportunities related to SSR.
